docs/docs/lib/node.mdx

```diff
@@ -210,17 +269,40 @@ configureCache({
 
 In order to run A/B tests, you need to set up a tracking callback function. This is called every time a user is put into an experiment and can be used to track the exposure event in your analytics system (Segment, Mixpanel, GA, etc.).
 
+You can specify this globally in your GrowthBookMultiUser class and/or per-request in your user context.
+
+We recommend using a global callback if you plan to track events directly from Node and using a user context callback if you plan to send events to the front-end to fire.
```

Comment:
I think this can be a bit clearer. I understand the difference approaches (maybe add link to the section explaining the diff between global/user context earlier in the docs), and I know you say more with deferred tracking below. However, I don't think it's entirely clear when the tracking cb will fire. It'd be helpful if you could give more guidance on when/under what conditions that happens.